frontend/src/components/common/SideNav.vue

```diff
@@ -93,23 +81,12 @@ const props = defineProps({
 </template>
 <style lang="scss" scoped>
 @import '@/assets/styles/styles.scss';
-
-.sidenav-logo {
-    cursor: pointer;
-    img {
-        margin: 0px 0px 1rem 0.188rem;
-        width: 8.75rem;
-        height: 2rem;
-    }
-}
-
 .sidenav {
     position: fixed;
-    padding: 0.75rem 0rem;
+    padding: 0.313rem 0rem;
```

Comment:
This is 5px to sum up with the 15px of the `li`, resulting in 20px requested in Sigma. It has been done this way because the margins between `li` elements look weird with 20px, so it has been kept with 15px.

Reply:
I think because we also have a primevue sidebar header element here, so the space doesn't look correct, let's check the the primevue sidebar component, if could remove the header. 
<img width="937" alt="Screen Shot 2024-04-17 at 11 22 43 AM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/e7ca6a66-240a-4b7d-874c-ff85070b8336">

Reply:
Sorry about that... I know what is happening. I did remove the primevue sidebar header in the common repo, that is why I do not see the space. I'll try to find a solution within our code, if not possible we would need to update the common repo.

Reply:
oh I see, yeah, we shouldn't update the common repo, thanks!!